src/Class.js

```diff
@@ -1,10 +1,10 @@
 var WebJack = {};
 
-(function(exports){
```

Comment:
Changed it like in the example, now it does not throw an error. 

I don't know if there is a better way, but I wonder why we need server compatibility?